changes/ce/fix-10577.en.md

```diff
@@ -0,0 +1 @@
+HTTP API to return status code `404` instead of `20X` if 'built_in_database' not configured as authorization source.
```

Comment:
nit:
Modified HTTP API behavior for APIs managing the `built_in_database` authorization source: They will now return a `404` status code if 'built_in_database' is not set as the authorization source, replacing the former `20X` response.